src/cart/crt_rpc.h

```diff
@@ -638,6 +638,10 @@ static inline bool
 crt_rpc_cb_customized(struct crt_context *crt_ctx,
 		      crt_rpc_t *rpc_pub)
 {
+	/* Do not use custom cb in case of SWIM. */
+	if (rpc_pub->cr_opc & 0xFE000000U)
```

Comment:
This check is not correct, as it will pass if cr_opc is for example 0x0200 0000, or for any opcode other than 0x0100 0000. Instead this should use either 0xFFFFFF mask or shift by 24 and compare remaining 8 bits

Reply:
ok